Maple2.Server.Game/Model/Field/Actor/FieldPlayer.cs

```diff
@@ -105,6 +106,13 @@ public class FieldPlayer : Actor<Player> {
             flagTick = (long) (tickCount + TimeSpan.FromSeconds(2).TotalMilliseconds);
         }
 
+        // do delayed NpcControl packet broadcast on field join
+        if (BroadcastNpcControlTick != 0 && tickCount >= BroadcastNpcControlTick) {
+            BroadcastNpcControlTick = 0;
+
+            Field.BroadcastNpcControl(this);
+        }
```

Comment:
Why is this in Player? And why is the tick only set once and never again?

Reply:
It's to send the first time npc control packet batch that has guaranteed sequences that arent -1, referring to "current animation". The bug was being caused by -1s being sent for sequence ids with no prior animation having ever been played on the client. I'll change the name of the function & variable to include "first" in the name